src/web/demo.rkt

```diff
@@ -282,7 +285,8 @@
      (define job-id (start-job command))
      (response/full 201 #"Job started" (current-seconds) #"text/plain"
                     (list (header #"Location" (string->bytes/utf-8 (url check-status job-id)))
-                          (header #"X-Job-Count" (string->bytes/utf-8 (~a (job-count)))))
+                          (header #"X-Job-Count" (string->bytes/utf-8 (~a (job-count))))
+                          (header #"X-Herbie-job-id" (string->bytes/utf-8 job-id)))
```

Comment:
It's traditional to capitalize, so `X-Herbie-Job-ID`. Though headers are case-insensitive, so it's not that important.

Reply:
Good to know thank you!